packages/loader/container-loader/src/container.ts

```diff
@@ -1312,7 +1317,11 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     }
 
     private async rehydrateDetachedFromSnapshot(detachedContainerSnapshot: ISummaryTree) {
-        const snapshotTree: ISnapshotTree = getSnapshotTreeFromSerializedContainer(detachedContainerSnapshot);
+        const blobs = new Map<string, ArrayBufferLike>();
+        const snapshotTree = getSnapshotTreeFromSerializedContainer(detachedContainerSnapshot, blobs);
```

Comment:
Given that blobs is the output, consider returning {snapshot, blobs} as opposed to creating it upfront and passing as arg.

Reply:
Done.